### PR TITLE
Add translations module and county codes for frontend

### PR DESCRIPTION
**Pull request by pbn4**
Addresses https://github.com/bloom-housing/bloom/issues/1097

### CODE REVIEW

File: backend/core/src/translations/entities/translation.entity.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { AbstractEntity } from "../../shared/entities/abstract.entity"
+import { Expose } from "class-transformer"
+import { IsEnum, IsString, MaxLength } from "class-validator"
+import { ValidationsGroupsEnum } from "../../shared/validations-groups.enum"
+import { ApiProperty } from "@nestjs/swagger"
+import { CountyCode } from "../../shared/types/county-code"
+import { Language } from "../../shared/types/language-enum"
+
+@Entity()
+@Index(["countyCode", "language", "key"], { unique: true })
```
Comment (software-project) [inline backend/core/src/translations/entities/translation.entity.ts:11]: I don't think we need `key` here. To get keys one by one, would require many requests to db. If we use just countyCode and language as keys and have a json field holding all the translations for county I think that would be enough. One call to get all of the needed translations.

**Review by software-project — CHANGES_REQUESTED** [review]
I like the enums refactoring. That's really nice. Translations I for language/ county I would make just as a 'bucket'. So basically one row in the db would hold sth like a whole json file on the frontend. I think that is going to be ok as I don't expect huge amount of keys per county.

File: backend/core/src/translations/entities/translation.entity.ts (hunk 1 above)
Comment (pbn4) [reply backend/core/src/translations/entities/translation.entity.ts]: Applied comments, thanks

File: backend/core/src/migration/1619094998058-alter-translations-table.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export class alterTranslationsTable1619094998058 implements MigrationInterface {
+    name = 'alterTranslationsTable1619094998058'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`DROP INDEX "IDX_4c17bc3d8eefdb8702bb24a2c5"`);
+        await queryRunner.query(`ALTER TABLE "translation" DROP COLUMN "key"`);
+        await queryRunner.query(`ALTER TABLE "translation" DROP COLUMN "text"`);
+        await queryRunner.query(`ALTER TABLE "translation" ADD "translations" jsonb NOT NULL`);
+        await queryRunner.query(`CREATE UNIQUE INDEX "IDX_ada354174d7f8a8f3d56c39bba" ON "translation" ("county_code", "language") `);
```
Comment (software-project) [inline backend/core/src/migration/1619094998058-alter-translations-table.ts:11]: I think we also need to add language to user_accounts so we know what language to use.

**Review by software-project — CHANGES_REQUESTED** [review]

File: backend/core/src/migration/1619094998058-alter-translations-table.ts (hunk 2 above)
Comment (pbn4) [reply backend/core/src/migration/1619094998058-alter-translations-table.ts:11]: It will be a part of a separate PR, too much stuff in those one already.

**Pull request merged**